specification/monitor/data-plane/Microsoft.Insights/preview/2023-03-01-preview/metricBatch.json

```diff
@@ -0,0 +1,449 @@
+        }
+      }
+    },
+    "Response": {
+      "description": "The metrics result for a resource.",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "timespan": {
+            "type": "string",
+            "description": "The timespan for which the data was retrieved. Its value consists of two datetimes concatenated, separated by '/'.  This may be adjusted in the future and returned back from what was originally requested."
+          },
+          "interval": {
+            "type": "string",
+            "format": "duration",
+            "description": "The interval (window size) for which the metric data was returned in.  This may be adjusted in the future and returned back from what was originally requested.  This is not present if a metadata request was made."
+          },
+          "namespace": {
+            "type": "string",
+            "description": "The namespace of the metrics been queried"
+          },
+          "resourceregion": {
+            "type": "string",
+            "description": "The region of the resource been queried for metrics."
+          },
+          "resourceid": {
+            "type": "string",
+            "description": "The resource that has been queried for metrics."
+          },
+          "value": {
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/Metric"
+            },
+            "description": "the value of the collection."
+          }
+        },
+        "required": [
+          "timespan",
+          "value"
+        ]
+      }
+    },
+    "MetricBatchBody": {
+      "description": "List of resource ids to fetch metric namespaces for",
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "example": [
+        "/subscriptions/123456728-abcd-1234-abcd-123456789abc/resourceGroups/TestGroup/providers/Microsoft.Storage/storageAccounts/TestStorage1",
+        "/subscriptions/123456728-abcd-1234-abcd-123456789abc/resourceGroups/TestGroup/providers/Microsoft.Storage/storageAccounts/TestStorage2"
+      ]
+    },
+    "LocalizableString": {
+      "type": "object",
+      "required": [
+        "value"
+      ],
+      "properties": {
+        "value": {
+          "type": "string",
+          "description": "the invariant value."
+        },
+        "localizedValue": {
+          "type": "string",
+          "description": "the locale specific value."
+        }
+      },
+      "description": "The localizable string class."
+    },
+    "Unit": {
+      "type": "string",
+      "description": "the unit of the metric.",
+      "enum": [
+        "Count",
+        "Bytes",
+        "Seconds",
+        "CountPerSecond",
+        "BytesPerSecond",
+        "Percent",
+        "MilliSeconds",
+        "ByteSeconds",
+        "Unspecified",
+        "Cores",
+        "MilliCores",
+        "NanoCores",
+        "BitsPerSecond"
+      ],
+      "x-ms-enum": {
```

Comment:
You can include "values" in the "x-ms-enum" with a description for each value, and these descriptions will then be shown in the generated REST API Docs. Descriptions for enum values will improve the developer experience for your service. See the [autorest documentation][] for the syntax of the "values" field in "x-ms-enum".

[autorest documentation]: https://github.com/Azure/autorest/tree/main/docs/extensions#x-ms-enum